scripts/generate-react-wrappers.ts

```diff
@@ -0,0 +1,6 @@
+import { generateReactWrappers } from '@patternfly/pfe-tools/react/generate-wrappers.js';
```

Comment:
`@lit-labs/react` got graduated to `@lit/react` with lit 3.0.0. we'll do that update separately i think